cluster/src/main/java/org/apache/iotdb/cluster/log/catchup/LogCatchUpTask.java

```diff
@@ -39,17 +43,25 @@
 public class LogCatchUpTask implements Runnable {
 
   private static final Logger logger = LoggerFactory.getLogger(LogCatchUpTask.class);
-
-  private List<Log> logs;
+  private static final int LOG_NUM_IN_BATCH = 128;
   Node node;
   RaftMember raftMember;
+  private List<Log> logs;
+  private boolean useBatch = false;
 
   public LogCatchUpTask(List<Log> logs, Node node, RaftMember raftMember) {
     this.logs = logs;
     this.node = node;
     this.raftMember = raftMember;
   }
 
+  public LogCatchUpTask(List<Log> logs, Node node, RaftMember raftMember, boolean useBatch) {
```

Comment:
I think maybe it's better to make `useBatch` a configuration item instead of calling it manually every time

Reply:
Sure, I will fix it